opendbc/car/tesla/carstate.py

```diff
@@ -136,10 +142,12 @@ def get_can_parsers(CP):
       ("DAS_control", 25),
       ("DAS_steeringControl", 50),
       ("DAS_status", 2),
-      ("DAS_settings", 2),
       ("SCCM_steeringAngleSensor", 100),
     ]
 
+    if is3Y:
+      ap_party_messages.append(("DAS_settings", 2))
+
```

Comment:
```suggestion
    if CP.carFingerprint in (CAR.TESLA_MODEL_3, CAR.TESLA_MODEL_Y):
      ap_party_messages.append(("DAS_settings", 2))

```

no need to abstract into a variable for something this simple

Reply:
need to import CAR from values at the top